test/docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       - mongodb
 
   agent:
-    container_name: kontena-agent
+    container_name: e2e-kontena-agent
```

Comment:
Nope, just something I fiddled with and seemed to commit by accident, will revert